readthedocs/proxito/views/hosting.py

```diff
@@ -510,11 +520,20 @@ def _v1(self, project, version, build, filename, url, request):
                         "trigger": "Slash",  # Could be something like "Ctrl + D"
                     },
                 },
+                "filetreediff": {
+                    "enabled": False,
+                },
```

Comment:
I'm a little confused where we're setting `enabled: False` in the code. Is it just the default and then we're overriding it if we're enabling it?

Reply:
Yeah, we default to false, and then check if the project supports it, or if the version allows it, and if the user has permissions.